wxt_cidrbot/git_webhook_handler.py

```diff
@@ -100,6 +99,112 @@ def webhook_request(self, event):
 
                 self.Api.messages.create(self.room_id, markdown=message_uninstall)
 
+        elif x_event_type in ('issues', 'pull_request'):
+            if event_action == 'opened':
+                self.triage_issue(installation_id, json_string, x_event_type)
+
+    # Assign a new github issue/pr on webhook
+    def triage_issue(self, installation_id, json_string, x_event_type):
+        event_info = self.check_installation(installation_id)
+        issue_type = "Pull request"
+        query_key = "pull_request"
+
+        # Issues and prs have different dict structures
+        if x_event_type == 'issues':
+            issue_type = "Issue"
+            query_key = "issue"
+            issue_num = json_string['issue']['number']
+        else:
+            issue_num = json_string['number']
+
+        room_id = event_info[0]['room_id']
+        issue_title = json_string[query_key]['title']
+        issue_url = json_string[query_key]['url']
+        issue_user = json_string[query_key]['user']['login']
+        repo_name = json_string['repository']['full_name']
+        repo_url = json_string['repository']['html_url']
+
+        hyperlink_format = f'<a href="{issue_url}">{issue_title}</a>'
+        hyperlink_format_repo = f'<a href="{repo_url}">{repo_name}</a>'
+        message = f"{issue_type} {hyperlink_format} created in {hyperlink_format_repo}. Performing automated triage:"
+
+        try:
```

Comment:
move this block to right after you get room_id - no reason to do all that other stuff if you're not going to triage anything

Reply:
Moved this to the correct place